lib/performance/AssetsOverSizeLimitWarning.js

```diff
@@ -2,23 +2,16 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Sean Larkin @thelarkinn
 */
-var SizeFormatHelpers = require("../SizeFormatHelpers");
+"use strict";
+const SizeFormatHelpers = require("../SizeFormatHelpers");
 
-function AssetsOverSizeLimitWarning(assetsOverSizeLimit, assetLimit) {
-	Error.call(this);
-	Error.captureStackTrace(this, AssetsOverSizeLimitWarning);
-	this.name = "AssetsOverSizeLimitWarning";
-	this.assets = assetsOverSizeLimit;
-
-	var assetLists = this.assets.map(function(asset) {
-		return "\n  " + asset.name + " (" + SizeFormatHelpers.formatSize(asset.size) + ")";
-	}).join("");
-
-	this.message = "asset size limit: The following asset(s) exceed the recommended size limit (" + SizeFormatHelpers.formatSize(assetLimit) + "). \n" +
-		"This can impact web performance.\n" +
-		"Assets: " + assetLists;
-}
-module.exports = AssetsOverSizeLimitWarning;
-
-AssetsOverSizeLimitWarning.prototype = Object.create(Error.prototype);
-AssetsOverSizeLimitWarning.prototype.constructor = AssetsOverSizeLimitWarning;
+module.exports = class AssetsOverSizeLimitWarning extends Error {
+	constructor(assetsOverSizeLimit, assetLimit) {
+		super();
+		Error.captureStackTrace(assetsOverSizeLimit, assetLimit);
+		this.name = "AssetsOverSizeLimitWarning";
+		this.assets = assetsOverSizeLimit;
+		const assetLists = this.assets.map(asset => `\n  ${asset.name} (${SizeFormatHelpers.formatSize(asset.size)})`).join();
+		this.message = `asset size limit: The following asset(s) exceed the recommended size limit (${SizeFormatHelpers.formatSize(assetLimit)}). \nThis can impact web performance.\nAssets: ${assetLists}`;
```

Comment:
Now I remember why I did this.  That space between `(${SizeFormatHelpers.formatSize(assetLimit)}).` and `\nThis` can't be seen and can get cut off by some word processors, but it's absence causes tests to fail.